packages/frontend/src/components/Search/FilterSearchData.tsx

```diff
@@ -0,0 +1,121 @@
+import { flatMapByKey, generateYearsBetween } from '../../utils/helpers';
+import categoryData from '../../assets/data/aineistoluokka.json';
+
+export enum ItemTypeEnum {
+  CHECKBOX = 'checkbox',
+}
+
+export interface IItem {
+  name: string;
+  type?: ItemTypeEnum;
+  items?: IItem[];
+}
+
+export interface IFilterSearchData {
+  name: string;
+  type?: ItemTypeEnum;
+  items: IItem[];
+}
+
+// TODO: should this be hardcoded?
+// TODO: this is temporary, possible change in data format
+export const FilterSearchData: IFilterSearchData[] = [
+  {
+    name: 'Muoto',
+    items: [
+      {
+        type: ItemTypeEnum.CHECKBOX,
+        name: 'PDF',
+      },
+      {
+        type: ItemTypeEnum.CHECKBOX,
+        name: 'Excel',
+      },
+      {
+        type: ItemTypeEnum.CHECKBOX,
+        name: 'Image',
+      },
+      {
+        type: ItemTypeEnum.CHECKBOX,
+        name: 'PowerPoint',
+      },
+      {
+        type: ItemTypeEnum.CHECKBOX,
+        name: 'Word',
+      },
+    ],
+  },
+  {
+    name: 'Aika',
```

Comment:
These are definitely something that need to be created dynamically up to the current year. Some helper function that gets (currentYear, 1980) as parameters?

Reply:
Do you mean we take start year as of 1980 and end year as this current year, and divide by 10 years which result in 5 group of years? (Currently have only helper `generateYearsBetween` that intakes start and end year :/ which is indeed more hard-coding)

Reply:
Yeah. One option that'll last for 7 years is to have `'2020-2022'` be replaced in a way that the end date is current year. But come 2030, this solution would no longer hold. Someone remembering to do this during the next 7 years if not done now does sounds a bit unlikely as well 😄